bin/dev/install_tools.sh

```diff
@@ -1,90 +1,54 @@
 #!/bin/bash
 set -o errexit -o nounset
 
-# Get version information
-. "$(dirname "$0")/versions.sh"
+# Get version information and set destination paths
+. "$(dirname "$0")/../common/versions.sh"
+
+if id -u vagrant >& /dev/null; then
+  SCF_BIN_DIR="${SCF_BIN_DIR:-/usr/local/bin}"
```

Comment:
We can, but then all users of the .box will also have to do `export SCF_BIN_DIR=/usr/local/bin` before running this script ( and the one in `bin/common/..`) I'm fine with either one, but setting it here means I don't have to remember to do anything later (for the CI deployment)